### PR TITLE
Fixes #13580: Use parameter that speak for themselves in centreon plugin

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/13580

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: centreon/bin/centreon-plugin [hunk 1]
```diff
@@ -303,17 +299,19 @@ def applyRudderMonitoringConfigurations(conf):
         print("[!] Unable to connect to Centreon webservice. Check centreon.conf ?")
         sys.exit(1)
 
+    # for debug only first-half of synchronize-hosts
     if(args['pull']):
```
Comment (VinceMacBuche) [inline centreon/bin/centreon-plugin]: I think you can remove "pull push and update" since theu are not documented anymore

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**